<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="15" hostname="WSAMZN-7ID4CD15" failures="76" tests="159" name="Remote Hands" time="83.765" errors="0" timestamp="2023-04-04T20:13:12 IST">
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.808">
    <skipped/>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.561">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.createRemoteHandsTicketAndVerifyEndToEndFlow(CreateRemoteHandsTicket.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.678">
    <skipped/>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.651">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.createRemoteHandsTicketAndVerifyEndToEndFlow(CreateRemoteHandsTicket.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.519">
    <skipped/>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketAndVerifyEndToEndFlow" time="0.49">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.createRemoteHandsTicketAndVerifyEndToEndFlow(CreateRemoteHandsTicket.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createRemoteHandsTicketAndVerifyEndToEndFlow -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketwithNotificationRecipient" time="0.285"/>
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketwithNotificationRecipient" time="0.619">
    <skipped/>
  </testcase> <!-- createRemoteHandsTicketwithNotificationRecipient -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="createRemoteHandsTicketwithNotificationRecipient" time="0.506">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.createRemoteHandsTicketwithNotificationRecipient(CreateRemoteHandsTicket.java:170)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createRemoteHandsTicketwithNotificationRecipient -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRHTicketForInvalidUser" time="0.2"/>
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.498">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.492">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.523">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.486">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.468">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.487">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.469">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.504">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.477">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.502">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.538">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.495">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.523">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.494">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.512">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.474">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.476">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.472">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.488">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.52">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.48">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.492">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.538">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.484">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.487">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.55">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.254"/>
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.28"/>
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.479">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.477">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.543">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.483">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.487">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.487">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.531">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.466">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.482">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.481">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.257"/>
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.54">
    <skipped/>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CreateRemoteHandsTicket" name="verifyCreateRemoteHandsTicket" time="0.476">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CreateRemoteHandsTicket.verifyCreateRemoteHandsTicket(CreateRemoteHandsTicket.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreateRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getAllRemoteHandsTicketsInfo" time="0.537">
    <skipped/>
  </testcase> <!-- getAllRemoteHandsTicketsInfo -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getAllRemoteHandsTicketsInfo" time="0.548">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.getAllRemoteHandsTicketsInfo(GetRemoteHandsTicketDetails.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- getAllRemoteHandsTicketsInfo -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getCategoryListForRemoteHandsRequestType" time="0.469">
    <skipped/>
  </testcase> <!-- getCategoryListForRemoteHandsRequestType -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getCategoryListForRemoteHandsRequestType" time="0.472">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.getCategoryListForRemoteHandsRequestType(GetRemoteHandsTicketDetails.java:143)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- getCategoryListForRemoteHandsRequestType -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getCategoryListForRemoteHandsRequestType" time="0.509">
    <skipped/>
  </testcase> <!-- getCategoryListForRemoteHandsRequestType -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getCategoryListForRemoteHandsRequestType" time="0.562">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.getCategoryListForRemoteHandsRequestType(GetRemoteHandsTicketDetails.java:143)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- getCategoryListForRemoteHandsRequestType -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="getCategoryListForRemoteHandsRequestType" time="0.264"/>
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.433">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.433">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.427">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.454">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.532">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.468">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.431">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.427">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.436">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.471">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.457">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.453">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.47">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.448">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.446">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.509">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.472">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="4.519">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.427">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.444">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.45">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.427">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.509">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.443">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.418">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.439">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.46">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.434">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.467">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.471">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.451">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.46">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.446">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.474">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.443">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.492">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.449">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.417">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.443">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.45">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.473">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.443">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.464">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.431">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.435">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.438">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.436">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.442">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.475">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.452">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.43">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.455">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.418">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.451">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.436">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.499">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.436">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.474">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.435">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.483">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.487">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.456">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.438">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.432">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.43">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.432">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.486">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.439">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.562">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.546">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.577">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.439">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.688">
    <skipped/>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetAllWithFilters" time="0.457">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetAllWithFilters(GetRemoteHandsTicketDetails.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetAllWithFilters -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetRemoteHandsTicketInfoByInvalidId" time="0.463">
    <skipped/>
  </testcase> <!-- verifyGetRemoteHandsTicketInfoByInvalidId -->
  <testcase classname="com.gapi.testCases.GetRemoteHandsTicketDetails" name="verifyGetRemoteHandsTicketInfoByInvalidId" time="0.431">
    <failure type="java.lang.AssertionError" message="expected [404] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [401]
at com.gapi.testCases.GetRemoteHandsTicketDetails.verifyGetRemoteHandsTicketInfoByInvalidId(GetRemoteHandsTicketDetails.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyGetRemoteHandsTicketInfoByInvalidId -->
  <testcase classname="com.gapi.testCases.EditRemoteHandsTicket" name="verifyEditCustomerReferenceOfAnExistingRemoteHandsTicket" time="0.486">
    <skipped/>
  </testcase> <!-- verifyEditCustomerReferenceOfAnExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.EditRemoteHandsTicket" name="verifyEditCustomerReferenceOfAnExistingRemoteHandsTicket" time="0.466">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.EditRemoteHandsTicket.verifyEditCustomerReferenceOfAnExistingRemoteHandsTicket(EditRemoteHandsTicket.java:98)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyEditCustomerReferenceOfAnExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.EditRemoteHandsTicket" name="verifyEditNotificationRecipientsOfAnExistingRemoteHandsTicket" time="0.519">
    <skipped/>
  </testcase> <!-- verifyEditNotificationRecipientsOfAnExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.EditRemoteHandsTicket" name="verifyEditNotificationRecipientsOfAnExistingRemoteHandsTicket" time="0.499">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.EditRemoteHandsTicket.verifyEditNotificationRecipientsOfAnExistingRemoteHandsTicket(EditRemoteHandsTicket.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyEditNotificationRecipientsOfAnExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CancelRemoteHandsTicket" name="verifyCancelRemoteHandsTicket" time="0.562">
    <skipped/>
  </testcase> <!-- verifyCancelRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CancelRemoteHandsTicket" name="verifyCancelRemoteHandsTicket" time="0.492">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.CancelRemoteHandsTicket.verifyCancelRemoteHandsTicket(CancelRemoteHandsTicket.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCancelRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.CancelRemoteHandsTicket" name="verifyCancelRemoteHandsTicketwithoutID" time="0.443">
    <skipped/>
  </testcase> <!-- verifyCancelRemoteHandsTicketwithoutID -->
  <testcase classname="com.gapi.testCases.CancelRemoteHandsTicket" name="verifyCancelRemoteHandsTicketwithoutID" time="0.451">
    <failure type="java.lang.AssertionError" message="expected [404] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [401]
at com.gapi.testCases.CancelRemoteHandsTicket.verifyCancelRemoteHandsTicketwithoutID(CancelRemoteHandsTicket.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyCancelRemoteHandsTicketwithoutID -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicket" time="0.462">
    <skipped/>
  </testcase> <!-- AddUsercommentsRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicket" time="0.472">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.AddUsercommentsRemoteHandsTicket(AddCommentsToTheRemoteHandsTicket.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- AddUsercommentsRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicket" time="0.481">
    <skipped/>
  </testcase> <!-- AddUsercommentsRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicket" time="0.493">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.AddUsercommentsRemoteHandsTicket(AddCommentsToTheRemoteHandsTicket.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- AddUsercommentsRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicketwithoutMandatoryParameters" time="0.466">
    <skipped/>
  </testcase> <!-- AddUsercommentsRemoteHandsTicketwithoutMandatoryParameters -->
  <testcase classname="com.gapi.testCases.AddCommentsToTheRemoteHandsTicket" name="AddUsercommentsRemoteHandsTicketwithoutMandatoryParameters" time="0.493">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.AddUsercommentsRemoteHandsTicketwithoutMandatoryParameters(AddCommentsToTheRemoteHandsTicket.java:172)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- AddUsercommentsRemoteHandsTicketwithoutMandatoryParameters -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="AddAttachmentstoRemoteHandsTicket" time="0.548">
    <skipped/>
  </testcase> <!-- AddAttachmentstoRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="AddAttachmentstoRemoteHandsTicket" time="0.486">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.gapi.utilities.TestUtilities.getWONumberFromResponse(TestUtilities.java:490)
at com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket.AddAttachmentstoRemoteHandsTicket(AddAttachmentsToTheRemoteHandsTicket.java:215)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AddAttachmentstoRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="AddAttachmentstoRemoteHandsTicket" time="0.49">
    <skipped/>
  </testcase> <!-- AddAttachmentstoRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="AddAttachmentstoRemoteHandsTicket" time="0.506">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.gapi.utilities.TestUtilities.getWONumberFromResponse(TestUtilities.java:490)
at com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket.AddAttachmentstoRemoteHandsTicket(AddAttachmentsToTheRemoteHandsTicket.java:215)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AddAttachmentstoRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="addAttachmentstoRemoteHandsTicketWithOutRequestIDAndfilename" time="0.248"/>
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="getAttachmentOfRemoteHandsTicketByID" time="0.507">
    <skipped/>
  </testcase> <!-- getAttachmentOfRemoteHandsTicketByID -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="getAttachmentOfRemoteHandsTicketByID" time="0.475">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.gapi.utilities.TestUtilities.getWONumberFromResponse(TestUtilities.java:490)
at com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket.getAttachmentOfRemoteHandsTicketByID(AddAttachmentsToTheRemoteHandsTicket.java:270)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- getAttachmentOfRemoteHandsTicketByID -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="uploadAttachmentToTheExistingRemoteHandsTicket" time="0.516">
    <skipped/>
  </testcase> <!-- uploadAttachmentToTheExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="uploadAttachmentToTheExistingRemoteHandsTicket" time="0.484">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket.uploadAttachmentToTheExistingRemoteHandsTicket(AddAttachmentsToTheRemoteHandsTicket.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- uploadAttachmentToTheExistingRemoteHandsTicket -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="uploadEmptyTextDocumentToTheValidRemoteHandsRequest" time="0.631">
    <skipped/>
  </testcase> <!-- uploadEmptyTextDocumentToTheValidRemoteHandsRequest -->
  <testcase classname="com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket" name="uploadEmptyTextDocumentToTheValidRemoteHandsRequest" time="0.498">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at com.gapi.testCases.AddAttachmentsToTheRemoteHandsTicket.uploadEmptyTextDocumentToTheValidRemoteHandsRequest(AddAttachmentsToTheRemoteHandsTicket.java:179)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- uploadEmptyTextDocumentToTheValidRemoteHandsRequest -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeSuite generateAuthorizationToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite generateAuthorizationToken -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeSuite generateAuthorizationToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite generateAuthorizationToken -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeSuite generateAuthorizationToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite generateAuthorizationToken -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeSuite generateAuthorizationToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite generateAuthorizationToken -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeSuite generateAuthorizationToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite generateAuthorizationToken -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeTest BeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest BeforeSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeTest BeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest BeforeSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeTest BeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest BeforeSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeTest BeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest BeforeSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@BeforeTest BeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest BeforeSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@AfterTest AfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest AfterSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@AfterTest AfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest AfterSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@AfterTest AfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest AfterSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@AfterTest AfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest AfterSuite -->
  <testcase classname="com.gapi.testCases.BaseClass" name="@AfterTest AfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest AfterSuite -->
</testsuite> <!-- Remote Hands -->
